packages/ringcentral-integration/modules/CallMonitor/index.js

```diff
@@ -362,17 +367,26 @@ export default class CallMonitor extends RcModule {
           } else {
             lastCalleeType = calleeTypes.unknow;
           }
-        } else if (_lastCallInfo.calleeType) {
+        } else if (
+          _fromSessionId === fromSessionId
+          && _lastCallInfo.calleeType
```

Comment:
then because if we start it with `null`, then there is chance this would throw a type error: can not read property calleeType on null